lib/presentation/views/details_view.dart

```diff
@@ -0,0 +1,94 @@
+          title: const Text('LINK DEVELOPMENT',
+              style: TextStyle(color: Colors.white)),
+          backgroundColor: Colors.black,
+          iconTheme: const IconThemeData(color: Colors.white),
+          actions: [
+            IconButton(
+              icon: const ImageIcon(AssetImage('assets/images/search.png')),
+              color: Colors.white,
+              onPressed: () {},
+            ),
+          ],
+        ),
+        backgroundColor: Colors.grey[200],
+        body: SingleChildScrollView(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Padding(
+                padding: EdgeInsets.all(size * 0.05),
+                child: Container(
+                  color: Colors.white,
+                  child: Column(
+                    children: [
+                      Container(
+                          width: size * 0.9,
+                          height: size * 0.5,
+                          child: Image.network(
+                            article.urlToImage,
+                          )),
+                      Padding(
+                        padding: EdgeInsets.all(size * 0.03),
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: [
+                            Text(article.title,
+                                style: TextStyle(
+                                    fontSize: size * 0.05,
+                                    fontWeight: FontWeight.w300)),
+                            Text('By ${article.author}',
+                                style: TextStyle(
+                                    fontSize: size * 0.03, color: Colors.grey)),
+                            Text(article.description,
+                                style: TextStyle(
+                                    fontSize: size * 0.03, color: Colors.grey))
+                          ],
+                        ),
+                      )
+                    ],
+                  ),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: ElevatedButton(
+                  style: ButtonStyle(
+                    shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+                        RoundedRectangleBorder(
+                      borderRadius: BorderRadius.zero,
+                    )),
+                    backgroundColor: MaterialStateProperty.all(Colors.black),
+                  ),
+                  onPressed: () {
+                    _launchUrl(Uri.parse(article.url));
+                  },
+                  child: Text('OPEN WEBSITE',
+                      style: TextStyle(
+                          fontSize: size * 0.04, color: Colors.white, fontWeight: FontWeight.w300)),
+                ),
+              )
+            ],
+          ),
```

Comment:
the button not like the design

<img width="370" alt="image" src="https://github.com/user-attachments/assets/3f461eda-2080-4c82-b18d-86d5a2b4be90">